### PR TITLE
204 Add `LDOSEDTM` to `adae`

### LINKED ISSUE #204: Add `LDOSEDTM` to `adae`
Used in AEL02_ED template.

`LDOSEDTM` = End Date/Time of Last Dose
- Datetime should occur before `ASTDTM`

### PR DESCRIPTION
**Pull request by edelarua**
Closes #204

### CODE REVIEW

File: R/radae.R [hunk 1]
```diff
@@ -164,6 +164,13 @@ radae <- function(ADSL, # nolint
       origin = "1970-01-01"
     )) %>%
     dplyr::mutate(AENDY = ceiling(as.numeric(difftime(.data$AENDTM, .data$TRTSDTM, units = "days")))) %>%
+    dplyr::mutate(LDOSEDTM = as.POSIXct(
+      ifelse(.data$TRTSDTM < .data$ASTDTM,
+        lubridate::as_datetime(runif(1, .data$TRTSDTM, .data$ASTDTM), tz = "EST"),
```
Comment (shajoezhu) [inline R/radae.R:169]: hi @edelarua , I was wondering if this is the timezone we use everywhere?

Comment (edelarua) [reply R/radae.R:169]: @shajoezhu yes, all of the dates in this dataset use the Eastern time zone.

Comment (shajoezhu) [reply R/radae.R:169]: Excellent! Thanks for checking this @edelarua

**Review by shajoezhu — APPROVED** [review]
lgtm!

**Pull request merged**